src/skill_tree.md

```diff
@@ -84,8 +84,7 @@ href = "https://github.com/rust-lang/const-eval/issues/14"
 
 [[group]]
 name = "offset_of"
-label = "offset_of\nfeature:const_ptr_offset"
-href = "https://github.com/rust-lang/rust/issues/71499"
```

Comment:
`offset_of` has very little to do with `offset`/`wrapping_offset`. If we want the latter in the skill tree, they should be a separate node.